libdnf/repo/repo.cpp

```diff
@@ -452,8 +452,6 @@ void Repo::load_available_repo(LoadFlags flags) {
         solv_repo->load_repo_ext(RepodataType::PRESTO, *downloader.get());
     }
 
-    // updateinfo must come *after* all other extensions, as it is not a real
-    // extension, but contains a new set of packages
```

Comment:
I think this comment wasn't actually true, it's just that previously updateinfo had to come after main. Now it should work even if we loaded updateinfo after the main solvables. That's not what we do now, though we do have the `LoadFlags` argument to `Repo::load_available_repo()` and in theory one could load a repo with just comps or updateinfo data.